content/resources/user/submit-data/sower.md

```diff
@@ -0,0 +1,119 @@
+```
+ssh -i ~/.ssh/my_key_pair_here.pem onekgenomes@ec2-23-20-189-94.compute-1.amazonaws.com
+```
+
+- To look through the bucket and find the data files of interest, you first need to install the client [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html). After successful installation, AWS CLI needs to be [configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html). Insert the location and information from your keypair:
+    ```
+    $/usr/local/bin: aws configure
+    AWS Access Key ID [*******************2]:
+    AWS Secret Access Key [*******************H]:
+    Default region name [None]: us-east-1
+    Default output format [None]:
+    ```
+
+- To access the bucket structure and find files, run:
+    ```
+    aws s3 ls s3//:bucketname
+    ```
+
+- To copy files to your local environment run `aws s3 cp s3://1000genomes/[path-to-file] [local-path]`:
+    ```
+    aws s3 cp s3://1000genomes/sequence_indices/20091216_20100311.stats.csv /usr/local/bin/sower_test/
+    ```
+    More documentation about AWS CLI terminal commands can be found [here](https://aws.amazon.com/cli/#file_commands_anchor).
+
+- After finding the path to files of interest in the bucket, calculating their size and md5, the manifest can be created. In our example, the access is open, so an asterisk can be inserted into the `acl` column:
+
+    ```
+    GUID	url	size	md5	acl
+     	s3://1000genomes/sequence_indices/20091216_20100311.stats.csv	745	613ccfbc68287db60c663519dab6a4ff	['*']
+    ```
+
+    This file will be now indexed by clicking "Index Files" in the [UI](https://gen3.datacommons.io/indexing).
+
+- The resulting manifest including the GUID can be downloaded as `.log` after clicking "Download Manifest":
+    ```
+    guid	urls	size	md5	acl
+    c146a966-7617-4f2f-b628-57a5f11de879	s3://1000genomes/sequence_indices/20091216_20100311.stats.csv	745	613ccfbc68287db60c663519dab6a4ff	['*']
+    ```
+
+- The submission files for the metadata can be now prepared following the instructions described in the [data submission](https://gen3.org/resources/user/submit-data/#4-submit-additional-project-metadata) or briefly described below. For this example we will use Windmill for the submission and show the steps as described as option 1 above, since it demonstrates the ability to link to a file node that is deep within the graph model.
```

Comment:
I wouldn't mention the product name "Windmill". I would just say "data portal" or "data commons website".

Reply:
done